docs/measuring-experiments/traffic.md

```diff
@@ -1 +1,13 @@
 # Traffic and Traffic Inbalances
+
+The details page for each experiment contains a Set Up page where you can configure the % of traffic you want to randomize into the experiment.
```

Comment:
Let's try to follow the Google developer documentation formatting guide here:

https://developers.google.com/style/text-formatting

so in particular, you'll want to bold **Set Up** page because it's a UI element.

I think the one exception is that we can continue to use Headline case rather than sentence case in the headings and subheadings